telegram/ext/_commandhandler.py

```diff
@@ -71,31 +71,35 @@ class CommandHandler(Handler[Update, CCT]):
         ValueError: when command is too long or has illegal chars.
 
     Attributes:
-        command (:obj:`str` | Tuple[:obj:`str`] | List[:obj:`str`]):
-            The command or list of commands this handler should listen for.
+        commands (Set[:obj:`str`]):
+            The set of command(s) this handler should listen for.
```

Comment:
the property already has a docstring, so you can remove this documentation of the attribute